src/app/period/period.component.html

```diff
@@ -19,7 +19,7 @@ <h4 class="mat-h4" [class.warning]="committedAllocationsTooHigh()">{{committedAl
         <button mat-fab (click)="disableEditing()" color="primary"><mat-icon>done</mat-icon></button>
     </span>
     <span *ngIf="!isEditingEnabled">
-        <button mat-fab (click)="enableEditing()" color="primary"><mat-icon>edit</mat-icon></button>
+        <button mat-fab [disabled]="!userHasEditPermissions" (click)="enableEditing()" color="primary"><mat-icon>edit</mat-icon></button>
```

Comment:
This is super nice and will really help the user experience. Thanks for adding this!

[No action needed]